terraform/modules/redis_prometheus_exporter/resource.tf

```
@@ -0,0 +1,32 @@
+resource cloudfoundry_service_key redis-key {
+  name             = "redis-key"
+  service_instance = var.redis_service_instance_id
+}
+
+locals {
+  url = "rediss://${cloudfoundry_service_key.redis-key.credentials.host}:${cloudfoundry_service_key.redis-key.credentials.port}"
```

I think you could just use `uri` (think its available in the credentials object)
```suggestion
  url = cloudfoundry_service_key.redis-key.credentials.uri"
```
and set its value to `REDIS_ADDR` and not set the password explicitly.

---

changing it to that method caused 
`2020-11-17T11:58:42.65+0000 [APP/PROC/WEB/0] ERR time="2020-11-17T11:58:42Z" level=error msg="Couldn't connect to redis instance"`